### PR TITLE
Implement JS API, part 1: rewritable units (#38)

### PR DESCRIPTION
**Pull request by inikulin**

### CODE REVIEW

**Comment by sejoker** [comment]
do you plan to add tests using https://rustwasm.github.io/docs/wasm-bindgen/wasm-bindgen-test/browsers.html as a part of this PR? It's hard to know whether API are covers everything we need without any kind of tests.

**Comment by inikulin** [comment]
@sejoker Yes. This is just part one, which can't really be tested in the browser yet as there is no way to manually create rewritable units.

Just wanted to do that in parts, otherwise the PR would be huge. Tests will be added once I implement bindings for the rewriter itself.

File: js-api/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "lol-html-js-api"
+version = "0.1.0"
+authors = ["Ivan Nikulin <inikulin@cloudflare.com>"]
+edition = "2018"
+
+[dependencies]
+lol_html = { path = "../" }
+serde = { version="1.0.104", features = ["derive"] }
```
Comment (xtuc) [inline js-api/Cargo.toml]: nit
```suggestion
serde = { version = "1.0.104", features = ["derive"] }
```

Comment (inikulin) [reply js-api/Cargo.toml]: Thanks!

**Pull request merged**